types/parse-commit-message/index.d.ts

```diff
@@ -2,27 +2,27 @@ declare module 'parse-commit-message' {
   let parser: {
     validate(commits: string[] | string, ret: boolean): boolean;
     validateCommit(commit: object, ret: boolean): boolean;
-    parseCommit(commit: string): Commit;
+    parseCommit(commit: string): ParseCommitMessageCommit;
   };
   export = parser;
 }
 
-type Header = {
+type ParseCommitMessageHeader = {
   type: string;
   scope?: string | null;
   subject: string;
 };
 
-type Commit = {
-  header: Header;
+type ParseCommitMessageCommit = {
+  header: ParseCommitMessageHeader;
```

Comment:
```suggestion
  header: ParsedCommitMessageHeader;
```

Reply:
No, it wouldn't match the installed library.